libs/merkle_proof/README.md

```diff
@@ -24,22 +24,25 @@ Once imported, using the Merkle Proof library is as simple as calling the desire
 
 ## Using the Merkle Proof Library in Fuels-rs
 
-To generate a Merkle Tree and corresponding proof for your Sway Smart Contract, use the [Fuel-Merkle](https://github.com/FuelLabs/fuel-merkle) crate. 
+To generate a Merkle Tree and corresponding proof for your Sway Smart Contract, use the [Fuel-Merkle](https://github.com/FuelLabs/fuel-vm/tree/master/fuel-merkle) crate. 

```

Comment:
Should we be pointing at `master` here or point at a specific version and then keep updating?

Reply:
I'd rather just point at master and let the devs decide if they need an older version themselves. Constantly updating this would be a pain and probably forgotten about.